tools/src/test/java/com/nedap/archie/archetypevalidator/ArchetypeValidatorTest.java

```diff
@@ -233,6 +233,27 @@ public void parentWithDifferentRmRelease() throws IOException, ADLParseException
         }
     }
 
+    @Test
```

Comment:
Is this the right place for the test and test files?

Reply:
Yes, it is. Maybe we also need a test that fails?